

Tanium: 6.5.314.4301, OS: Ubuntu 14.04 trusty, Python: 2.7.6
========================================================================================

This is the output from running test/test_pytan_valid_server_tests.py against the following:

* PyTan Version: 2.1.0
* Tanium Platform Version: 6.5.314.4301
* Test Date: 2015:09:04 21:26:30 PDT
* OS Version running PyTan: Ubuntu 14.04 trusty
* Python version running PyTan: 2.7.6 (default, Mar 22 2014, 22:59:56) 
* Output from tests: `pytan_validation_test-6.5.314.4301-Ubuntu_14.04_trusty-2.7.6.log <../_static/valid_pytan_tests/pytan_validation_test-6.5.314.4301-Ubuntu_14.04_trusty-2.7.6.log>`_

